neuro_san/http_sidecar/handlers/health_check_handler.py

```diff
@@ -22,6 +23,17 @@ class HealthCheckHandler(RequestHandler):
     Handler class for API endpoint health check.
     """
 
+    def initialize(self):
+        """
+        This method is called by Tornado framework to allow
+        injecting service-specific data into local handler context.
+        Here we use it to inject CORS headers if so configured.
+        """
+        if os.environ.get("AGENT_ALLOW_CORS_HEADERS") is not None:
```

Comment:
We could consider reversing the default here so we allow CORS unless it's disabled, since in almost all cases we'd want it.